### PR TITLE
[FX-1735] Featured artist view

### PR DESCRIPTION
**Pull request by xtina-starr**
Addresses: [FX-1735](https://artsyproduct.atlassian.net/browse/FX-1735)

**Issue:**
When there are several featured artists within a collection and a user clicks the "View more" link, the featured artist module pushes the content beneath the featured artist module down making it not visible to the user.

**Fix:**
This PR refactors the featured artist component to take the user to a new view when the "View all" link is clicked. This displays all the artists that are within the collection and gives the user a back button to navigate back to the collection.

featured artists list

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>
<b>🃏 FAIL</b> in <a href="https://github.com/artsy/eigen/blob/featured-artist-view/src/lib/Scenes/Collection/__tests__/Collection-tests.tsx">Collection-tests.tsx</a>


<li>
<a href='https://github.com/artsy/eigen/blob/featured-artist-view/src/lib/Scenes/Collection/__tests__/Collection-tests.tsx#L46'>renders a snapshot</a>
<br/>
Snapshot name: `Collection renders a snapshot 1`- Snapshot+ Received@@ -9,11 +9,11 @@ ListHeaderComponent={ <ForwardRef(Relay(CollectionHeader)) collection={ Object { "__fragmentOwner": Object {- "identifier": "bc3659ca34d0a3d9e4807e2294e185e2{}",+ "identifier": "97a30e65f961be03f0e817ca03d683d2{}", "node": Object { "fragment": Object { "argumentDefinitions": Array [], "kind": "Fragment", "metadata": null,@@ -570,11 +570,11 @@ "alias": null, "args": Array [ Object { "kind": "Literal", "name": "size",- "value": 9,+ "value": 4, }, ], "concreteType": "Artist", "kind": "LinkedField", "name": "merchandisableArtists",@@ -667,11 +667,11 @@ }, ], "storageKey": null, }, ],- "storageKey": "merchandisableArtists(size:9)",+ "storageKey": "merchandisableArtists(size:4)", }, Object { "alias": null, "args": null, "kind": "ScalarField",@@ -717,11 +717,11 @@ "storageKey": "marketingCollection(slug:"doesn't matter")", }, ], }, "params": Object {- "id": "bc3659ca34d0a3d9e4807e2294e185e2",+ "id": "97a30e65f961be03f0e817ca03d683d2", "metadata": Object { "relayTestingSelectionTypeInfo": Object { "marketingCollection": Object { "enumValues": null, "nullable": true,

<details>
<summary>Full message</summary>
<pre><code>
Error: expect(received).toMatchSnapshot()

Snapshot name: `Collection renders a snapshot 1`

- Snapshot
+ Received

@@ -9,11 +9,11 @@
      ListHeaderComponent={
        <ForwardRef(Relay(CollectionHeader))
          collection={
            Object {
              "__fragmentOwner": Object {
-               "identifier": "bc3659ca34d0a3d9e4807e2294e185e2{}",
+               "identifier": "97a30e65f961be03f0e817ca03d683d2{}",
                "node": Object {
                  "fragment": Object {
                    "argumentDefinitions": Array [],
                    "kind": "Fragment",
                    "metadata": null,
@@ -570,11 +570,11 @@
                                "alias": null,
                                "args": Array [
                                  Object {
                                    "kind": "Literal",
                                    "name": "size",
-                                   "value": 9,
+                                   "value": 4,
                                  },
                                ],
                                "concreteType": "Artist",
                                "kind": "LinkedField",
                                "name": "merchandisableArtists",
@@ -667,11 +667,11 @@
                                      },
                                    ],
                                    "storageKey": null,
                                  },
                                ],
-                               "storageKey": "merchandisableArtists(size:9)",
+                               "storageKey": "merchandisableArtists(size:4)",
                              },
                              Object {
                                "alias": null,
                                "args": null,
                                "kind": "ScalarField",
@@ -717,11 +717,11 @@
                        "storageKey": "marketingCollection(slug:"doesn't matter")",
                      },
                    ],
                  },
                  "params": Object {
-                   "id": "bc3659ca34d0a3d9e4807e2294e185e2",
+                   "id": "97a30e65f961be03f0e817ca03d683d2",
                    "metadata": Object {
                      "relayTestingSelectionTypeInfo": Object {
                        "marketingCollection": Object {
                          "enumValues": null,
                          "nullable": true,
    at Object.toMatchSnapshot (/Users/travis/build/artsy/eigen/src/lib/Scenes/Collection/__tests__/Collection-tests.tsx:46:31)
    at Object.asyncJestTest (/Users/travis/build/artsy/eigen/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
    at /Users/travis/build/artsy/eigen/node_modules/jest-jasmine2/build/queueRunner.js:43:12
    at new Promise (<anonymous>)
    at mapper (/Users/travis/build/artsy/eigen/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
    at /Users/travis/build/artsy/eigen/node_modules/jest-jasmine2/build/queueRunner.js:73:41
</code></pre>
</details>
</li>
<br/>

</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>
<b>🃏 FAIL</b> in <a href="https://github.com/artsy/eigen/blob/featured-artist-view/src/lib/Scenes/Collection/Components/__tests__/FullFeaturedArtistList-tests.tsx">FullFeaturedArtistList-tests.tsx</a>


<li>
<a href='https://github.com/artsy/eigen/blob/featured-artist-view/src/lib/Scenes/Collection/Components/__tests__/FullFeaturedArtistList-tests.tsx#L33'>renders properly</a>
<br/>
Snapshot name: `FullFeaturedArtistList renders properly 1`- Snapshot+ Received- "<rctscrollview contentcontainerstyle="[object Object]" data="[object Object],[object Object],[object Object],[object Object],[object Object]" initialnumtorender="10" maxtorenderperbatch="10" scrolleventthrottle="50" updatecellsbatchingperiod="50" windowsize="21" numcolumns="1" viewabilityconfigcallbackpairs="" stickyheaderindices=""><view><view><view mb="2"><text font-family="Unica77LL-Medium" font-size="16px" lineheight="26px" textalign="center">Feature Artists</text></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Pablo Picasso</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, 1877-1973</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="secondaryOutline" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#000">Following</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Andy Warhol</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, 1947-1987</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="secondaryOutline" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#000">Following</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Joan Miro</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">Spanish, 1877-1983</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="secondaryOutline" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#000">Following</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Jean-Michel Basquiat</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, 1960-2001</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="primaryBlack" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#FFF">Follow</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Kenny Scharf</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, b. 1958</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="primaryBlack" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#FFF">Follow</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view></view></rctscrollview>"+ "<rctscrollview contentcontainerstyle="[object Object]" data="[object Object],[object Object],[object Object],[object Object],[object Object]" initialnumtorender="10" maxtorenderperbatch="10" scrolleventthrottle="50" updatecellsbatchingperiod="50" windowsize="21" numcolumns="1" viewabilityconfigcallbackpairs="" stickyheaderindices=""><view><view><view mb="2"><text font-family="Unica77LL-Medium" font-size="16px" lineheight="26px" textalign="center">Featured Artists</text></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Pablo Picasso</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, 1877-1973</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="secondaryOutline" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#000">Following</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Andy Warhol</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, 1947-1987</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="secondaryOutline" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#000">Following</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Joan Miro</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">Spanish, 1877-1983</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="secondaryOutline" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#000">Following</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Jean-Michel Basquiat</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, 1960-2001</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="primaryBlack" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#FFF">Follow</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Kenny Scharf</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, b. 1958</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="primaryBlack" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#FFF">Follow</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view></view></rctscrollview>"

<details>
<summary>Full message</summary>
<pre><code>
Error: expect(received).toMatchSnapshot()

Snapshot name: `FullFeaturedArtistList renders properly 1`

- Snapshot
+ Received

- "<rctscrollview contentcontainerstyle="[object Object]" data="[object Object],[object Object],[object Object],[object Object],[object Object]" initialnumtorender="10" maxtorenderperbatch="10" scrolleventthrottle="50" updatecellsbatchingperiod="50" windowsize="21" numcolumns="1" viewabilityconfigcallbackpairs="" stickyheaderindices=""><view><view><view mb="2"><text font-family="Unica77LL-Medium" font-size="16px" lineheight="26px" textalign="center">Feature Artists</text></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Pablo Picasso</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, 1877-1973</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="secondaryOutline" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#000">Following</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Andy Warhol</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, 1947-1987</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="secondaryOutline" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#000">Following</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Joan Miro</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">Spanish, 1877-1983</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="secondaryOutline" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#000">Following</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Jean-Michel Basquiat</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, 1960-2001</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="primaryBlack" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#FFF">Follow</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Kenny Scharf</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, b. 1958</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="primaryBlack" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#FFF">Follow</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view></view></rctscrollview>"
+ "<rctscrollview contentcontainerstyle="[object Object]" data="[object Object],[object Object],[object Object],[object Object],[object Object]" initialnumtorender="10" maxtorenderperbatch="10" scrolleventthrottle="50" updatecellsbatchingperiod="50" windowsize="21" numcolumns="1" viewabilityconfigcallbackpairs="" stickyheaderindices=""><view><view><view mb="2"><text font-family="Unica77LL-Medium" font-size="16px" lineheight="26px" textalign="center">Featured Artists</text></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Pablo Picasso</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, 1877-1973</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="secondaryOutline" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#000">Following</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Andy Warhol</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, 1947-1987</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="secondaryOutline" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#000">Following</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Joan Miro</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">Spanish, 1877-1983</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="secondaryOutline" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#000">Following</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Jean-Michel Basquiat</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, 1960-2001</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="primaryBlack" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#FFF">Follow</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view><view><view width="100%" pb="20"><view flexdirection="row" justifycontent="space-between" flexwrap="nowrap" focusable="true"><view mr="1" justifycontent="center"><image resizemode="stretch" style="width: 45px; height: 45px; border-radius: 22.5px;" source="[object Object]"></image></view><view justifycontent="center" width="0" flexgrow="1"><text font-family="ReactNativeAGaramondPro-Regular" font-size="16px" lineheight="20px" ellipsizemode="tail" numberoflines="1" color="black100">Kenny Scharf</text><text font-family="Unica77LL-Regular" font-size="12px" lineheight="16px" ellipsizemode="tail" numberoflines="1" color="black60">American, b. 1958</text></view><view ml="1" flexdirection="row" alignitems="center" justifycontent="flex-end"><view flexdirection="row" focusable="true"><view variant="primaryBlack" theme="[object Object]" px="1"><view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" color="#FFF">Follow</text></view><text font-family="Unica77LL-Medium" font-size="12px" lineheight="16px" role="presentation">Following</text></view></view></view></view></view></view></view></rctscrollview>"
    at toMatchSnapshot (/Users/travis/build/artsy/eigen/src/lib/Scenes/Collection/Components/__tests__/FullFeaturedArtistList-tests.tsx:33:25)
    at tryCatch (/Users/travis/build/artsy/eigen/node_modules/regenerator-runtime/runtime.js:45:40)
    at Generator.invoke [as _invoke] (/Users/travis/build/artsy/eigen/node_modules/regenerator-runtime/runtime.js:271:22)
    at Generator.prototype.<computed> [as next] (/Users/travis/build/artsy/eigen/node_modules/regenerator-runtime/runtime.js:97:21)
    at tryCatch (/Users/travis/build/artsy/eigen/node_modules/regenerator-runtime/runtime.js:45:40)
    at invoke (/Users/travis/build/artsy/eigen/node_modules/regenerator-runtime/runtime.js:135:20)
    at /Users/travis/build/artsy/eigen/node_modules/regenerator-runtime/runtime.js:145:13
    at /Users/travis/build/artsy/eigen/node_modules/react-tracking/node_modules/core-js/modules/es.promise.js:113:22
    at flush (/Users/travis/build/artsy/eigen/node_modules/react-tracking/node_modules/core-js/internals/microtask.js:26:9)
    at processTicksAndRejections (internal/process/task_queues.js:75:11)
</code></pre>
</details>
</li>
<br/>

</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Scenes/Collection/Components/FeaturedArtists.tsx [hunk 1]
```diff
@@ -77,26 +69,27 @@ export class FeaturedArtists extends React.Component<FeaturedArtistsProps, Featu
...
+                  action_type: Schema.ActionTypes.Tap,
+                  action_name: Schema.ActionNames.ViewMore,
+                  context_screen: Schema.PageNames.Collection,
+                  context_module: Schema.ContextModules.FeaturedArtists,
+                  flow: Schema.Flow.FeaturedArtists,
+                })
+              }}
+            >
+              <ViewAll size="4">View all</ViewAll>
+              {/* <Box>
```
Comment (sweir27) [inline src/lib/Scenes/Collection/Components/FeaturedArtists.tsx]: Can we remove the commented-out bit?

Comment (xtina-starr) [reply src/lib/Scenes/Collection/Components/FeaturedArtists.tsx]: Oops! lol

File: src/lib/Scenes/Collection/Components/__tests__/FullFeaturedArtistList-tests.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import { FullFeaturedArtistListTestsQueryRawResponse } from "__generated__/FullFeaturedArtistListTestsQuery.graphql"
+import { mockTracking } from "lib/tests/mockTracking"
+import { renderRelayTree } from "lib/tests/renderRelayTree"
+import React from "react"
+import { graphql } from "react-relay"
+import { FullFeaturedArtistListCollectionFixture } from "../__fixtures__/CollectionFixture"
+import { CollectionFeaturedArtistsContainer as CollectionFeaturedArtists } from "../FullFeaturedArtistList"
+jest.unmock("react-relay")
+
+describe("FullFeaturedArtistList", () => {
```
Comment (sweir27) [inline src/lib/Scenes/Collection/Components/__tests__/FullFeaturedArtistList-tests.tsx]: Nice tests!

**Pull request merged**